backend/src/controller/base_layers.rs

```diff
@@ -0,0 +1,61 @@
+//! `Plants` endpoints.
+
+use crate::{db::connection::Pool, service};
+use actix_web::{
+    delete, get, post,
+    web::{Data, Json, Path},
+    HttpResponse, Result,
+};
+
+use crate::model::dto::NewBaseLayerDto;
+
+/// Fetch a single [`BaseLayerDto`](crate::model::dto::BaseLayerDto) by its id.
+///
+/// # Errors
+/// * If the connection to the database could not be established.
+#[utoipa::path(
+    context_path = "/api/layers/base/{id}",
+    responses(
+        (status = 200, description = "Fetch a base layer by its id", body = BaseLayerDto),
+    )
+)]
+#[get("/{id}")]
+pub async fn find_by_id(id: Path<i32>, pool: Data<Pool>) -> Result<HttpResponse> {
+    let payload = service::base_layers::find_by_id(*id, &pool).await?;
+    Ok(HttpResponse::Ok().json(payload))
+}
+
+/// Publish a  [`NewBaseLayerDto`](crate::model::dto::NewBaseLayerDto).
+///
+/// # Errors
+/// * If the connection to the database could not be established.
+#[utoipa::path(
+    context_path = "/api/layers/base",
+    responses(
```

Comment:
Add the schema of the request body:
```suggestion
    request_body = NewBaseLayerDto,
    responses(
```